src/codeu/chat/util/PlayInfo.java

```diff
@@ -0,0 +1,80 @@
+package codeu.chat.util;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.*;
+
+/**
+ * Info to store information about an instance of a play
+ */
+public class PlayInfo {
```

Comment:
Use this general rule of thumb to decide whether a class is a util class: 
1. Are all it's functions static? If yes, it's probably a Util class.
2. Do you, at any point, need to instantiate an object of the class? If yes, this means your application logic depends on the 'state' of this object, and so this should be part of the model (and hence go into chat/common or chat/server). Only the static functions should be part of the util class.